routes/folders.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const express = require('express');
+const mongoose = require('mongoose');
+
+const Folder = require('../models/folder');
+const Note = require('../models/note');
+
+const router = express.Router();
+
+/* ========== GET/READ ALL ITEMS ========== */
+
+router.get('/', (req, res, next) => {
+
+  Folder.find()
+    .sort({ name : 'asc'})
```

Comment:
ascending is the default
```suggestion
    .sort('name')
```